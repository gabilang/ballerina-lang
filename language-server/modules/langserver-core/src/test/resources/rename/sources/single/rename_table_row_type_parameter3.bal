type Person record {|
    readonly int id;
    string name;
|};

function foo() {
    table<Person> tHuman = table [{id: 1, name: "Jane"}];
}
