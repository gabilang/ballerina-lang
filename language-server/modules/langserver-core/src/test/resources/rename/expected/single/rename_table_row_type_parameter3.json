{
  "source": {
    "file": "rename_table_row_type_parameter1.bal"
  },
  "position": {
    "line": 6,
    "character": 13
  },
  "prepareRename": {
    "valid": true
  },
  "result": {
    "changes": {
      "rename_table_row_type_parameter1.bal": [
        {
          "range": {
            "start": {
              "line": 0,
              "character": 5
            },
            "end": {
              "line": 0,
              "character": 11
            }
          },
          "newText": "Student"
        },
        {
          "range": {
            "start": {
              "line": 6,
              "character": 10
            },
            "end": {
              "line": 6,
              "character": 16
            }
          },
          "newText": "Student"
        }
      ]
    }
  }
}
